examples/segmentation/README.md

```diff
@@ -0,0 +1,36 @@
+# Semantic Segmentation Example
+
+This example illustrates how to train a semantic segmentation model in composer
+
```

Comment:
Maybe include a bit of installation instructions in the readme? Or, mention a docker image one can use.